builds/mesmer/power-chronomancer/index.md

```diff
@@ -73,35 +74,21 @@ Use the [Gear Optimizer](https://optimizer.discretize.eu/?m=fractals) to adjust
 <GridItem sm="7">
 ### Weapons Choices
 
-Always play Greatsword main-hand.
+Always use Greatsword in combat.
 
-<Grid>
-<GridItem sm="6">
-#### Main-hand
-**Dagger:**
-- strongest raw dps output
-- ranged cleave damage
+#### Off-hand
 
-**Sword:**
-- Auto-Attack chains boon rip
-- <Skill name="Blurred Frenzy"/> as a strong burst/evade
-- <Skill name="Swap"/> to leap at a target at max 600 range
+While Sword off=hand is your highest damage weapon choice, both Shield and Focus can be used situationally. It is recommended to run focus in trash heavy T4s as standard.
```

Comment:
It is recommended to use the Focus in to pull the trash mob.
Sword off-hand is your highest damage weapon choice.
Shield is situationally.